### DISCUSSION TITLE
Support for SDN

### DISCUSSION THREAD

**Discussion by Gramarin** [bpg/terraform-provider-proxmox#1565] [Q&A] [answered]
I currently don’t know what SDN is. I’ve exhausted all the resources that I’ve managed to come across. Before I go further down the rabbit hole, does bpg support SDN? I’m pretty new to Terraform and bpg, so I don’t know for sure. But searching this repo comes up with next to no hits. So I take it as no? I can just use either Proxmox firewalls or firewalls in each VM. So before I go learn something that I can’t even take advantage of as of right now I was just wondering on SDN status in bpg?

**Comment by bpg-dev** [comment, marked as answer]
SDN is not currently supported by the provider, see
- https://github.com/bpg/terraform-provider-proxmox/issues/817

You can find more info about SDN in this ticket.
